docker-compose.yaml

```diff
@@ -29,8 +29,8 @@ services:
     container_name: qdrant
     volumes:
       - qdrantstorage:/qdrant/storage
-    ports:
-      - "6333:6333"
+    # ports:
+    #   - "6333:6333"
```

Comment:
It might be considered dangerous to expose qdrant port outside - there are no built-in auth, so all update endpoints will be exposed